qmanager/policies/base/queue_policy_base.hpp

```diff
@@ -737,7 +737,7 @@ class queue_policy_base_t : public resource_model::queue_adapter_base_t
             }
         } else {
             bool found_in_prov = false;
-            if (erase_pending_job (job, found_in_prov) < 0)
+            if (erase_pending_job (job.get(), found_in_prov) < 0)
```

Comment:
RFC 7 states the C code "shall separate function names and the opening parenthesis": https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_7.html#c-coding-style-recommendations. Fluxion is almost all C++, but it follows the flux-core convention here.
```suggestion
            if (erase_pending_job (job.get (), found_in_prov) < 0)
```